src/index.html

```diff
@@ -29,17 +29,14 @@ <h4 style="display: flex; align-items: center; justify-content: center;">
         <h2>About</h2>
       </header>
       <p>
-        Apache Parquet is a powerful column-oriented data format, built from the ground up to as a modern 
-        alternative to CSV files. GeoParquet is an incubating <a href="https://ogc.org">Open Geospatial Consortium (OGC)</a>
-        standard that adds interoperable geospatial types (Point, Line, Polygon) to Parquet. 
+        Apache Parquet is a powerful column-oriented data format, built from the ground up to as a modern alternative to CSV files. GeoParquet is an incubating <a href="https://ogc.org">Open Geospatial Consortium (OGC)</a> standard that adds interoperable geospatial types (Point, Line, Polygon) to Parquet.
```

Comment:
I suppose putting everything on a single line is on purpose here? (since I don't see any other change) Is that needed for correct rendering? It does make looking at the diff harder ..

Reply:
I was making somewhat arbitrary changes here to confirm that the preview deploy worked.

Reply:
I also agree that the diff is horrible to read.  I had mistakenly thought that GitHub was soft-wrapping on HTML docs as well, but I guess those don't count as [prose](https://github.blog/2013-12-03-soft-wrapping-on-prose-diffs/).

I can make another PR that hard wrap the text here again.  I still do prefer to allow soft-wrapping on markdown docs as it avoids all the unnecessary noise when you re-wrap after adding a bit of text.

Reply:
Yes, sounds good, thanks!